api/services/call_log_service.py

```diff
@@ -74,3 +84,22 @@ def update_user_module_content_id_in_call_log(self, user_module_content_id):
             data = {}
             data["user_module_content_id"] = user_module_content_id
             self.update_call_logs(data)
+
+    def handle_parent_flow(self, jsonData):
+        parent_flow_data = {}
+        parent_flow_data["parent_flow_name"] = None
+        parent_flow_data["parent_flow_run_uuid"] = None
+        if "parent" in jsonData and "flow" in jsonData["parent"]:
+            parent_flow = jsonData["parent"]["flow"]
+            parent_flow_data["parent_flow_name"] = parent_flow["name"]
+            is_contains_missedcall_category = helpers.string_contains_list_item(
```

Comment:
Please rename `is_contains_missedcall_category` to `missedcall_category_list`

Reply:
Changed.